pkg/tsdb/cloudwatch/services/log_groups.go

```diff
@@ -51,3 +51,30 @@ func (s *LogGroupsService) GetLogGroups(req resources.LogGroupsRequest) ([]resou
 
 	return result, nil
 }
+
+func (s *LogGroupsService) GetLogGroupFields(request resources.LogGroupFieldsRequest) ([]resources.ResourceResponse[resources.LogGroupField], error) {
+	input := &cloudwatchlogs.GetLogGroupFieldsInput{
+		LogGroupName: aws.String(request.LogGroupName),
+	}
+	// we should use LogGroupIdentifier instead of LogGroupName, but currently the api doesn't accept LogGroupIdentifier. need to check if it's a bug or not.
+	// if request.LogGroupARN != "" Lo. need
```

Comment:
This means the GetLogGroupFields is not compatible for cross-account querying right now. Will bring this up with AWS, but don't think that should be preventing us from merging this pr to main.